perception/map_based_prediction/src/map_based_prediction_ros.cpp

```diff
@@ -557,7 +557,7 @@ void MapBasedPredictionROS::objectsCallback(
       "map",        // src
       rclcpp::Time(), rclcpp::Duration::from_seconds(0.1));
   } catch (tf2::TransformException & ex) {
-    RCLCPP_ERROR(get_logger(), ex.what());
+    RCLCPP_ERROR_STREAM(get_logger(), ex.what());
```

Comment:
`RCLCPP_ERROR(get_logger(), "%s", ex.what().c_str());` is better?

Reply:
if `RCLCPP_ERROR` is still used, your opinion is better.